spec/features/lifehacks/05_user_deletes_lifehack_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+feature 'user deletes lifehack', %{
+  As a User and the creator of the lifehack
+  I want to destroy the lifehack
+  So I can delete the content of the lifehack
+} do
+
+  # Acceptance Criteria:
+  # [] I must be able delete a lifehack from the lifehack show page
+  # [] If I am not the creator, I cannot delete the lifehack
+  # [] All associated reviews with the lifehack must also be deleted
+
+  let (:user) { FactoryGirl.create(:user) }
+  let!(:lifehack) { FactoryGirl.create(:lifehack, creator: user) }
+  let!(:lifehack2) { FactoryGirl.create(:lifehack, creator: user) }
+
+  let (:user2) { FactoryGirl.create(:user) }
+
+  scenario 'navigate to lifehack show page and delete lifehack' do
+    user_sign_in(user)
+    expect(page).to have_content(lifehack.title)
+    expect(page).to have_content(lifehack2.title)
+
+    click_link(lifehack.title)
+    expect(current_path).to eq(lifehack_path(lifehack))
+
+    click_button('Delete Lifehack')
+    expect(current_path).to eq(root_path)
+    expect(page).to have_content("Admin successfully deleted lifehack:
+    #{lifehack.title}")
+
+    expect(page).to have_content(lifehack2.title)
+  end
+
+  scenario 'user attempts to edit another user\'s lifehack' do
+    user_sign_in(user2)
+    expect(page).to have_content(lifehack2.title)
+
+    click_link(lifehack2.title)
+    click_button('Delete Lifehack')
+
+    expect(page).to have_content('You are not the Authorized User')
+    expect(current_path).to eq(root_path)
+    expect(page).to have_content(lifehack2.title)
+  end
+
+  scenario 'check to see if reviews are deleted with lifehack'
```

Comment:
It's better not to commit pending tests, just like it's better not to commit commented-out code. If you need to remember to do something, add it to whatever list you're using to keep track of your work elsewhere!